packages/block-library/src/query-title/edit.js

```diff
@@ -89,16 +93,35 @@ export default function QueryTitleEdit( {
 		titleElement = (
 			<>
 				<InspectorControls>
-					<PanelBody title={ __( 'Settings' ) }>
-						<ToggleControl
-							__nextHasNoMarginBottom
+					<ToolsPanel
+						label={ __( 'Settings' ) }
+						resetAll={ () =>
+							setAttributes( {
+								showPrefix: false,
```

Comment:
```suggestion
								showPrefix: true,
```

The default value of the `showPrefix` attribute is `true`.